### DISCUSSION TITLE
Queued maps length shown in multiplayer lobbies

### DISCUSSION THREAD

**Discussion by airone01** [ppy/osu#17729] [Ideas] [answered]
Recently I ended up managing a multiplayer lobby with around ~20 players. I found it hard to try and have maps from everyone playing equally, as i needed to open every map page and get its length to make sure we don’t play 6* 7’ marathons.

Is it possible to have a small label with the song length/time spent before the song plays for every song in the queue ?

I think somewhere around the right-side of the mapper’s name would be a good place.

**Comment by smoogipoo** [comment]
Probably something to be done in the [redesign](https://www.figma.com/file/XJCRjfvTlVIwZBWoAz5kB2/Client%2FPlaylist-Multiplayer?node-id=0%3A1), and for @arflyte to consider.

**Comment by arflyte** [comment, marked as answer]
Hmm, I'll try to put it somewhere

**Reply by smoogipoo** [reply]
We also need:
- The username or avatar of the user who added the playlist item.
- Maybe which freemods are allowed (the mods there are the required mods which every player must have)?

I think the rest (BPM, approach rate, other info that users might require that's present in osu-stable) can be added in a tooltip, I guess?

**Reply by airone01** [reply]
Mabe some of these can be additional, and shown only on hover, (the mods and the rest for example)

**Reply by Theighlin** [reply]
> I think the rest (BPM, approach rate, other info that users might require that's present in osu-stable) can be added in a tooltip, I guess?

As a player i'd really like for that information to be seen at a glance and not in a tooltip, at least for the next map in the playlist (it's shown in both song select and the tournament client if I'm not mistaken).
Players are used to it, and it's very likely that if not done this way, every now and then someone will forget to check it, or might be to lazy to do so every round, while it's probably in their interest to see it. (It was my original intent in opening #11306)

The other beatmaps info can perfectly be placed in a tooltip if they can't be fit in the new design. (it's new functionality not expected from those coming from stable)
